lib/rules/no-typos.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
     let propTypesPackageName = null;
     let reactPackageName = null;
 
-    function checkValidPropTypeQualfier(node) {
+    function checkValidPropTypeQualifier(node) {
```

Comment:
Someone put typos in `no-typos` so I can fix typos while I fix `no-typos`.

Reply:
![0ca36be035e2c8c79d6af4543c7a6990](https://user-images.githubusercontent.com/226640/41449394-b2ed0126-7015-11e8-895b-9835624758cd.jpg)